[lldb/crashlog] Create stackframes for non-crashed threads

medismailben opened the pull request:
By default, the crashlog script only loads images for the crashed thread as well as the application specific thread.

Prior to using SymbolFileJSON and ObjectFileJSON, it didn't make sense to create stackframes for the non-crashed threads however now, we should still create stackframes for non-crashed threads.

This should also let the user pull the images after the process is launched, using `add-dsym`.

rdar://129171513

JDevlieghere reviewed (approved):
LGTM but can we test this?

medismailben committed:
[lldb/crashlog] Create stackframes for non-crashed threads

By default, the crashlog script only loads images for the crashed
thread as well as the application specific thread.

Prior to using SymbolFileJSON and ObjectFileJSON, it didn't make sense
to create stackframes for the non-crashed threads however now, we should
still create stackframes for non-crashed threads.

This should also let the user pull the images after the process is
launched, using `add-dsym`.

rdar://129171513

Signed-off-by: Med Ismail Bennani <ismail@bennani.ma>
